### PR TITLE
Updates for RDM CD Downtime

### PR DESCRIPTION
**Pull request by Cephid**
This is just a small PR to adjust the values for the RDM Cooldown Downtime due to update of that module.  Also adjusted the first grace of CDs to the 3-8 standard opener.

### CODE REVIEW

File: src/parser/jobs/rdm/modules/GeneralCDDowntime.ts [hunk 1]
```diff
@@ -12,19 +12,19 @@ export default class GeneralCDDowntime extends CooldownDowntime {
 		// hope that it's accurate enough
 		{
```
Comment (AzariahAeroborn) [inline src/parser/jobs/rdm/modules/GeneralCDDowntime.ts]: Should this comment be changed or renewed in light of the code change?

**Comment by ayyaruq** [comment]
Looks fine to me but how common is that standard opener, is it optimal, or actually standard?

File: src/parser/jobs/rdm/modules/GeneralCDDowntime.ts (hunk 1 above)
Comment (Cephid) [reply src/parser/jobs/rdm/modules/GeneralCDDowntime.ts]: We're not actually sure honestly, we tried a multitude of tests but weren't entirely happy with any result.  This particular configuration yielded the most correct data across different situations.  We elected in the TC channel to just go with it and see how it goes with more real world data.

**Comment by Cephid** [comment]
It's the pinned opener, the TC channel also confirmed it for me when I was adjusting the old numbers.

**Pull request merged**